src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/image.yml

```diff
@@ -25,5 +25,18 @@ admin_products_v2_add_image:
             id_product: productId
```

Comment:
Remove this one too

Reply:
No this one was left on purpose, legacy parameters are used are not only used for url generation, they are also used to allow backward compatibility So that parameters from the Symfony request are injected into `Tools` so that the hook relying on `Tools::getValue('id_product')` can still get the data